src/views/Settings.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-title>{{ $t("Settings") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+    
+    <ion-content>
+      <ion-list>     
+        <!-- OMS information -->
+        <ion-item>
+          <ion-icon :icon="codeWorkingOutline" slot="start"/>
+          <ion-label>{{ $t("OMS") }}</ion-label>
+          <p slot="end">{{ baseURL ? baseURL : instanceUrl }}</p>
+        </ion-item>
+        <!-- Profile of user logged in -->
+        <ion-item>
+          <ion-icon :icon="personCircleOutline" slot="start" />
+          <ion-label>{{ userProfile !== null ? userProfile.partyName : '' }}</ion-label>
+          <ion-button slot="end" fill="outline" color="dark" @click="logout()">{{ $t("Logout") }}</ion-button>
+        </ion-item>
+      </ion-list>
+    </ion-content>
+  </ion-page>
+</template>
+
+<script lang="ts">
+import { IonButton, IonContent, IonHeader, IonIcon, IonItem, IonLabel, IonList, IonPage, IonTitle, IonToolbar, modalController } from '@ionic/vue';
+import { defineComponent } from 'vue';
+import { personCircleOutline, codeWorkingOutline } from 'ionicons/icons'
+import { mapGetters, useStore } from 'vuex';
+import { useRouter } from 'vue-router';
+
+export default defineComponent({
+  name: 'Settings',
+  components: {
+    IonButton, 
+    IonContent, 
+    IonHeader, 
+    IonIcon,
+    IonItem, 
+    IonLabel, 
+    IonList,
+    IonPage,  
+    IonTitle,
+    IonToolbar
+  },
+  data(){
+    return {
+      baseURL: process.env.VUE_APP_BASE_URL
```

Comment:
We will not store VUE_APP_BASE_URL in configuration in any case. We could remove this